content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -110,7 +110,7 @@ <h2 id="toc-into">Introduction</h2>
 on top of the system's foundational hardware capabilities.</p>
 <p>This tutorial introduces <a href="https://www.w3.org/TR/mediacapture-streams/"><code>navigator.mediaDevices.getUserMedia()</code></a>, which allows
 web apps to access a user's camera and microphone.</p>
-<h2 id="toc-history">The road to getUserMedia()</h2>
+<h2 id="toc-history">The road to <code>getUserMedia()</code> API</h2>
```

Comment:
Is house style for headings to use code font?

Nit: maybe 'the getUserMedia API'.

Reply:
We try to avoid using code items in headings, but if we use a code item in a heading, then we follow it with a descriptive noun. I'll add "the" in a moment.